### PR TITLE
Add Polar package

### PR DESCRIPTION
**Pull request by edaniels**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").

My package is a syntax highlighter for [Oso's Polar language](https://docs.osohq.com/any/reference/polar/polar-syntax.html).

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by edaniels** [comment]
@packagecontrol-bot request review?

**Comment by dennykorsukewitz** [comment]
> @dennykorsukewitz, how were you able to request the bot to do a review? Thanks!

A magician never reveals his tricks. 🪄 🎩 🐰  

But in this case it is because I have already created a PR that has been merged.

@dennykorsukewitz 
https://github.com/wbond/package_control_channel/pulls?q=is%3Apr+author%3Adennykorsukewitz+is%3Aclosed

@edaniels 
https://github.com/wbond/package_control_channel/pulls?q=is%3Apr+author%3Aedaniels+is%3Aclosed+

File: repository/p.json [hunk 1]
```diff
@@ -1838,6 +1838,19 @@
 				}
 			]
 		},
+		{
+			"name": "Polar",
+			"author": "Eric Daniels",
+			"details": "https://github.com/edaniels/sublime-polar",
+			"issues": "https://github.com/edaniels/sublime-polar/issues",
```
Comment (braver) [inline repository/p.json:1845]: You don't need to specify "issues" if it's just the default GitHub url.

**Comment by braver** [comment]
Thanks for your submission. The main thing missing from your syntax is that all your scopes should end with `.polar`. That way packages can easily target your specific scopes if they want to. Other than that, since you already built tests I'm sure this is superfluous, but let me mention the [Scope Naming guidelines](https://www.sublimetext.com/docs/scope_naming.html): please ensure your syntax follows what's outlined in there as much as possible.

Let me know when you've added the suffixes and tagged a new release so we can merge and ship this thing 👍🏻

**Comment by edaniels** [comment]
Hey @braver, thanks for the review. Just addressed your changes in https://github.com/edaniels/sublime-polar/releases/tag/v0.0.2!

**Pull request merged**